scripts/utility-wfrp4e.js

```diff
@@ -612,6 +612,11 @@ class WFRP_Utility
     if (chatData.rollMode === "blindroll") chatData["blind"] = true;
     else if (chatData.rollMode === "selfroll") chatData["whisper"] = [game.user];
 
+    if ( forceWhisper ) { // Final force !
+      chatData["speaker"] = ChatMessage.getSpeaker();
```

Comment:
Why is `getSpeaker()` necessary here? I tend to avoid this function if I can.

Reply:
I don't know, not my stuff :( @LeRatierBretonnien.
Moreover getSpeaker takes 1 parameter and this is not the case.
I can delete it if you think so.

Reply:
Oh sorry, I thought you added the forceWhisper condition